README.md

```diff
@@ -4,60 +4,55 @@
 
 Roslyn provides open-source C# and Visual Basic compilers with rich code analysis APIs.  It enables building code analysis tools with the same APIs that are used by Visual Studio.
 
-### Language Design Discussion
+### C# and Visual Basic Language Feature Suggestions
 
-We are now taking language feature discussion in other repositories:
-- https://github.com/dotnet/csharplang for C# specific issues
-- https://github.com/dotnet/vblang for VB-specific features
-- https://github.com/dotnet/csharplang for features that affect both languages
+If you want to suggest a new feature for the C# or Visual Basic languages go here:
+- [dotnet/csharplang](https://github.com/dotnet/csharplang) for C# specific issues
+- [dotnet/vblang](https://github.com/dotnet/vblang) for VB-specific features
+- [dotnet/csharplang](https://github.com/dotnet/csharplang) for features that affect both languages
 
-Discussion about the transition of language design to the new repos is at https://github.com/dotnet/roslyn/issues/18002.
+### Getting started with the Roslyn APIs
```

Comment:
I would put this much lower. In general I would have the README basically increase in contributor complexity as they scroll top to bottom. Dealing with our APIs is far more complex than building the repo. 

Basically optimize for new contributors. Existing contributors probably don't need this info or at least know where to go searching for it.

Reply:
yeah, I kept this in its original location but I agree this should be moved (or even removed)